ruby/vampires.rb

```diff
@@ -0,0 +1,61 @@
+# Vampire Detector Quiz
+
+# release 4 loop
+puts 'How many are we interviewing?'
+applicants = Integer(gets.chomp)
+
+until applicants.zero?
+
+  # The Quiz
+  puts "Welcome to Werewolf Inc. What's your name?"
+  name = gets.chomp
+  vampire = false
+  vampire = true if name == 'Drake Cula' || name == 'Tu Fang'
```

Comment:
`vampire = (name == 'Drake Cula' || name == 'Tu Fang')`

This will return true or false by using the boolean logic rather then specifying the `true..if`. This makes `vampire` either equal true or false, rather then true or nil.